<launch>
    <node pkg="smarthome" type="gateway.js" name="gateway" output="screen"/>
    <node pkg="smarthome" type="device_manager" name="mgr" output="screen"/>
    <node pkg="smarthome" type="simple_light" name="light" output="screen"/>
    <node pkg="smarthome" type="simple_light" name="light_2" output="screen"/>
</launch>